report/01_keymode_report.Rmd

```diff
@@ -15,15 +15,20 @@ Is the popularity of songs in a major key different than songs in a minor key? (
 
 First of all, in music, "keys" are sets of notes which sound harmonious together. One of the most distinguishing features of a musical key is its "mode" which we can categorize as "major" or "minor." These two modes affect the mood of music similarly to how certain beats may make songs more "likeable." Music written in a major key-mode usually sounds happy, while music written in a minor key-mode usually sounds sad or serious. We are interested in whether these modes (major vs. minor) of songs' keys affect their popularity.
 
+Spotify is a music streaming platform. They have become an important platform as they currently have 191 million users. 87 million of these users, pay in order to be premium users. Their Top 100 is defined by the songs that are streamed the most throught a year. 
+
 Through this evaluation, we go through Spotify's Top 100 2017 data set to analyze whether people are actively listening to more songs that are in major key-modes or minor key-modes. 
 
 Once we load the data, we make a small summary to see actually how many songs in Spotify have a minor or a major mode. <br>
 
-Table01
-```{r, echo = FALSE}
-summary <- read.csv("../data/summary_data.csv")
-summary
-```
+Table01: Summary of how many songs have a major or minor mode
+
+
+| Mode | Avg Rank | Count | Diff. Est. |
```

Comment:
The instructions imply that the is meant to generated from script output, not manually.  I think Tiffany posted something in Slack last week about formatting the tables, but it was optional.